ui/imports/shared/stores/RootStore.qml

```diff
@@ -222,6 +222,12 @@ QtObject {
         return currencyStore.getFiatValue(balance, cryptoSymbol, fiatSymbol)
     }
 
+    function formatGweiAmount(feeCurrency) {
```

Comment:
hmm wondering if this shouldn't be a special case inside `currencyAmountToLocaleString`, so we ensure any place where we want to display an amount in `Gwei` will get the same formatting,